### PR TITLE
Horizon Prisoner Admin Ghost Role

### PR DESCRIPTION
**Pull request by Eyeveri**
Adds a ghost role for CCIA to enable, based upon request from the CCIA staff team.

### CODE REVIEW

File: code/modules/ghostroles/spawner/human/admin/ccia.dm [hunk 1]
```diff
@@ -124,6 +124,25 @@
...
 	mob_name_pick_message = "Pick a name."
 
+// Horizon Prisoner
+/datum/ghostspawner/human/admin/corporate/horizonprisoner
+	short_name = "prisoner"
+	name = "SCCV Horizon Prisoner"
+	desc = "You are a prisoner being held within the brig aboard the SCCV Horizon. This role requires special staff authorization to be used. You are not an antagonist, self antagonist rules still apply to you."
+
+	landmark_name = "HorizonPrisoner"
+	possible_species = list(SPECIES_HUMAN, SPECIES_SKRELL, SPECIES_SKRELL_AXIORI, SPECIES_IPC, SPECIES_UNATHI)
```
Comment (Greenjoe12345) [inline code/modules/ghostroles/spawner/human/admin/ccia.dm:134]: shouldn't it have the other species too?

Comment (Eyeveri) [reply code/modules/ghostroles/spawner/human/admin/ccia.dm:134]: Oh. I forgot to add the others, lol. I meant to go back to that, hold on

**Comment by Greenjoe12345** [comment]
Another thing that is an issue is the fact that special things for the character can't be set up in the ghost role customization screen
things such as markings, prosthetic limbs/organs and flavor text, along with augments, though i imagine in the case of a number of augments they'd also possibly need a augment disruptor.

**Comment by Eyeveri** [comment]
> Another thing that is an issue is the fact that special things for the character can't be set up in the ghost role customization screen things such as markings, prosthetic limbs/organs and flavor text, along with augments, though i imagine in the case of a number of augments they'd also possibly need a augment disruptor.

As admins, we cannot do this anyway. So when people have asked to have this setup for them in game, we just spawn a human mob and manually do this. The point of this spawner is to make it so that we do not have to interrupt our rounds everytime. This isn't just a ghost role to be played, it requires admin/ccia permission.

**Comment by Eyeveri** [comment]
!wip

**Comment by Arrow768** [comment]
For the ccia recorder to properly work, the correct character is needs to be set on the character that is being interviewed.
This is fundamentally incompatible with ghost spawners as ghost spawners we're never created with the intention of using them with persistent characters.

**Pull request closed without merging**